wiki/Placeholders.md

```diff
@@ -3965,11 +3965,13 @@ Per Player/Event placeholders:
 ----
 
 - ### **[ThemePark](https://www.spigotmc.org/resources/48648/)**
-> NO DOWNLOAD COMMAND
+> [Download here](https://api.extendedclip.com/expansions/themepark/)
```

Comment:
Doesn't matter if it is yet approved or not. Just provide the command itself. People will be able to download it through multiple ways.
```suggestion
> /papi ecloud download themepark
```